### PR TITLE
Feature/create default plan configuration

### PR DESCRIPTION
**Pull request by Franco0712**
## Description:
Add a default Plan for the organizations, it gets the default plan and its configuration from the one created in the migrations.
Blank=True is not set so the user cannot choose the "None" plan option.
This PR should be merged after the create-default-configuration PR because it creates the default plans in the migrations that will be used here
## Screenshots (if appropriate):
image

### CODE REVIEW

File: organization/models/organization.py [hunk 1]
```diff
@@ -79,33 +78,42 @@ class Meta:
...
 
 
 @receiver(post_save, sender=Organization, dispatch_uid='org_saved')
 def org_post_save_receiver(sender, instance, created, **kwargs):
-    """
+    '''
     Create a new AWS S3 bucket after that a new organization is saved.
-    """
+    '''
+    print('Instance')
```
Comment (fpyoung) [inline organization/models/organization.py]: is this log necessary?

**Review by fpyoung — CHANGES_REQUESTED** [review]

**Review by fpyoung — APPROVED** [review]

File: organization/models/organization.py [hunk 2]
```diff
@@ -79,23 +78,29 @@ class Meta:
...
+    '''
     Get from services trusted AWS S3 bucket name for a new organization.
-    """
+    '''
     # assign trusted bucket name
     instance.bucket_name = s3.generate_bucket_name(instance.name)
 
     aws_account = instance.aws_account
-    if instance.security_enabled and not aws_account.cf_private_key and not aws_account.cf_key_pair_id:
+    if (
```
Comment (NoeliaBentancor) [inline organization/models/organization.py]: Why are we changing this?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,142 +0,0 @@
-# Video Headline - Contributing
```
Comment (NoeliaBentancor) [inline CONTRIBUTING.md:1]: Why are we deleting this?

**Review by fpyoung — APPROVED** [review]
I re-tested the changes and everything works fine!

**Review by capuajuan — APPROVED** [review]
All good

**Pull request merged**